spec/order_spec.rb

```diff
@@ -19,10 +18,10 @@
 
   describe '#order_confirmation' do
     it 'confirms the order' do
-      menu = Menu.new
       subject.place_order(:chips, :lamb_shish, :nil)
```

Comment:
Great workaround with ":nil"! Could you make an array or somewhere to put the order they choose so that you can add more food items than three?